File: spring-data-requery-java/src/test/java/com/coupang/springframework/data/requery/java/domain/model/AddressTypeConverter.java
package com.coupang.springframework.data.requery.java.domain.model;

import io.requery.converter.EnumStringConverter;

/**
 * com.coupang.springframework.data.requery.java.domain.model.AddressTypeConverter
 *
 * @author debop
 * @since 18. 6. 4
 */
public class AddressTypeConverter extends EnumStringConverter<AddressType> {

    public AddressTypeConverter() {
        super(AddressType.class);
    }

}

File: src/test/kotlin/com/coupang/springframework/data/requery/domain/functional/AddressTypeConverter.kt
package com.coupang.springframework.data.requery.domain.functional

import io.requery.converter.EnumStringConverter

/**
 * AddressTypeConverter
 *
 * @author debop@coupang.com
 * @since 18. 5. 31
 */
class AddressTypeConverter: EnumStringConverter<AddressType>(AddressType::class.java)